### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by anirudhsundar98**

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+}
+
+#main-body-container::after {
+  content: "";
+  display: block;
+  height: 1.4em;
+}
+
+.navbar-nav > li, .navbar-nav > li > a {
+  transition: all 0.2s;
```
Comment (gauthamk97) [inline css/base.css]: This is causing some crazy purple jumpy shit on the nav bar. Only add transitions to the property you're changing (background).

File: css/members.css [hunk 2]
```diff
@@ -161,15 +157,3 @@ but underneath basic-details. Thus, the inner-details div will be visible but it
...
-@keyframes fadeInUp {
-    from {
-        opacity: 0;
-        transform: translateY(10px);
-    }
-    to {
-        opacity: 1;
-        transform: translateY(0px);
-    }
-}
```
Comment (gauthamk97) [inline css/members.css]: Undo **ALL** fadeIn and fadeOut changes you made. The old version seemed more responsive. The simplifications you made to `members.css` is okay though

File: css/main.css [hunk 3]
```diff
@@ -54,6 +54,10 @@
 #whatWeDoImage {
   margin: 10px 0px;
 }
+
+p.content-section-1 {
```
Comment (gauthamk97) [inline css/main.css]: Change class name to `justify-content`

**Review by gauthamk97 — CHANGES_REQUESTED** [review]
- Weird navbar purple jumpy shit
- Remove the animation changes you'd made to members page
- Class name change

File: css/base.css (hunk 1 above)
Comment (anirudhsundar98) [reply css/base.css]: Done

File: css/members.css (hunk 2 above)
Comment (anirudhsundar98) [reply css/members.css]: Done

File: css/main.css (hunk 3 above)
Comment (anirudhsundar98) [reply css/main.css]: Assuming these divs need to be styled further in the future, `section-1-content` is a better class name to add styles to than `justify-content`.

**Review by gauthamk97 — APPROVED** [review]

**Pull request merged**